CMakeLists.txt

```diff
@@ -50,18 +50,29 @@ set(CTR_DRBG_128_BIT_KEY_WARNING "${WARNING_BORDER}"
                          "${WARNING_BORDER}")
 
 # Python 3 is only needed here to check for configuration warnings.
-find_package(PythonInterp 3)
-if(PYTHONINTERP_FOUND)
+if(NOT CMAKE_VERSION VERSION_LESS 3.15.0)
```

Comment:
Please update the commit message accordingly, 3.15.0 and not 3.12.0.

Reply:
done